example_configs/dolibarr.md

```diff
@@ -0,0 +1,88 @@
+# Configuration pour Dolibarr
+
+## This example will help you to create user in dolibarr from your users in your lldap server from a specific group and to login with the password from the lldap server.
+
+## to connect ldap->dolibarr 
+
+Install module LDAP from Home -> Modules/Applications
+Go to the configuration of this module and fill it like this:
+
+
+Users and groups synchronization: LDAP -> Dolibarr
```

Comment:
Can you put the values (here and below) between ticks `` ? To make them stand out and show that that's what the user should enter.